src/plugins/map/MQ2Map.cpp

```diff
@@ -42,6 +42,8 @@ bool HighlightPulseIncreasing = true;
 int HighlightPulseIndex = 0;
 int HighlightPulseDiff = HighlightSIDELEN / 10;
 extern MapObject* gpActiveMapObjects;
+std::vector<MapFilterOption*> mapfilterObjectOptions;
+std::vector<MapFilterOption*> mapFilterOptions;
```

Comment:
since they aren't extern, I might suggest prefixing with `s_` to indicate that these are static maps. Also kind of weird that one variable is mapfilter and the other is mapFilter

Reply:
true, should be s_ and static if thats the case.

Reply:
I am actually wrong, I just found where it's extern'ed later and used in other implementation files.

Reply:
good point dan, and the latter part was a typo that I didn't notice.

Reply:
oh, these are extern, does that change your naming suggestion?